package.json

```diff
@@ -5,8 +5,12 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "tsc": "tsc"
```

Comment:
change the name of this script to be "build": "tsc". It does not matter what you label this attribute. it can build it can be tsc it can be anything you want. If a new dev comes in that knows nodejs he will recognize the word build as something that builds the application.